ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/traffic-policies.md

```diff
@@ -100,19 +101,36 @@ interface Port-Channel2
 
 | Match set | Type | Sources | Destinations | Protocol | Source Port(s) | Destination port(s) | Action |
 | --------- | ---- | ------- | ------------ | -------- | -------------- | ------------------- | ------ |
-| BLUE-C1-POLICY-01 | ipv4 | 10.0.0.0/8<br/>192.168.0.0/16 | DEMO-01 | tcp | 1,10-20 | ANY | action: PASS<br/>traffic-class: 5 |
+| BLUE-C1-POLICY-01 | ipv4 | 10.0.0.0/8<br/>192.168.0.0/16 | DEMO-01 | tcp<br/>udp | 1,10-20<br/>SERVICE-DEMO | ANY | action: PASS<br/>traffic-class: 5 |
 | BLUE-C1-POLICY-02 | ipv4 | DEMO-01<br/>DEMO-02 | ANY | tcp<br/>icmp | ANY | SERVICE-DEMO | action: PASS<br/>counter: DEMO-TRAFFIC<br/>dscp marking: 60 |
 | BLUE-C1-POLICY-03 | ipv4 | DEMO-01 | ANY | icmp | ANY | ANY | action: DROP<br/>counter: DROP-PACKETS<br/>logging |
 | BLUE-C1-POLICY-04 | ipv4 | DEMO-02 | DEMO-01 | tcp<br/>icmp | 22 | ANY | action: PASS<br/>traffic-class: 5 |
 | BLUE-C1-POLICY-05 | ipv4 | DEMO-02 | DEMO-01 | tcp | ANY | ANY | action: PASS<br/>traffic-class: 5 |
+| BLUE-C1-POLICY-06 | ipv4 | ANY | ANY | neighbors<br/>ip<br/>udp<br/>tcp<br/>icmp | 22<br/>SERVICE-DEMO | 1,10-20 | action: PASS |
```

Comment:
![Uploading Screenshot 2024-08-02 at 11.24.14.png…]()
Hmmm that's not really understandable here what corresponds to what - wondering if we should not change the way we render this (maybe one column that says:

```
<PROTOCOL> ( <SOURCE_PORT>, <DEST_PORT>)
```
which could be clearer

@ClausHolbechArista

Reply:
updated